src/backend/api/meals.js

```diff
@@ -12,103 +13,157 @@ router.get("/", async(request, response) => {
     }
 });
 
-//Returns all meals
-router.get("/titles", async(request, response) => {
+router.get("/", async(request, response) => {
     try {
+        //console.log(meals)
         // knex syntax for selecting things. Look up the documentation for knex for further info
-        const titles = await knex("meals").select("title");
-        response.json(titles);
+        const meals = await knex("meals");
+        response.json(meals);
     } catch (error) {
         throw error;
     }
 });
```

Comment:
since you implement get method with all filters in line 96 and it includes title filter line 122 
and  in the same function if there is no filters you will return the all meals in line 159
so we can delete these two functions here from lines 7 to 25